README.md

```diff
@@ -1,14 +1,58 @@
 ﻿# 🪡 jNeedle (or Needle)
+
 Yet another jar malware detection tool
 
 ---
+
+## Quickstart / How to use
+
+If you came from [MMPA](https://blog.mmpa.info) or already know what this program does, look here to see how to use jNeedle.
+
+### GUI (Graphical) mode
+
+The easiest way to use jNeedle is to use the GUI version.
+Just download the `jneedle-gui-xxx.jar` from the [release page](https://github.com/KosmX/jneedle/releases/latest) and double click on it.
+
+_Help, it won't start!_
+
+There currently is a bug in the GUI version with Java 1.8, which won't let the program start. This is known and will be fixed.  
+If you still want to use it, look at the [CLI mode](#cli-mode).

```

Comment:
Right now there are multiple issues with java 1.8 affecting the scanning. As soon as I can fix those I'll edit this, until then 1.8 is not working at all (or with limited performance)

Reply:
Will update that

Reply:
Fixed